components/support/error/src/lib.rs

```diff
@@ -1,60 +1,66 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 /// Define a wrapper around the the provided ErrorKind type.
 /// See also `define_error` which is more likely to be what you want.
+
 #[macro_export]
 macro_rules! define_error_wrapper {
```

Comment:
@thomcc Do we still need these wrappers? I would have thought `thiserror` takes care of this for us.